src/components/Notifications/Notifications.tsx

```diff
@@ -242,6 +244,14 @@ class NotificationManager {
         for (let k in this.boards_to_move_on) {
             board_ids.push(parseInt(this.boards_to_move_on[k].id));
         }
+
+        for (let k in this.active_boards) {
+            if (this.boards_to_move_on.length !== 0) {
+                return;
```

Comment:
If there are moves to play, do nothing (leave the function without advancing to the next game)

Reply:
Yeah I had a feeling the problem was in this area, just hadn't had the chance to sit down and read through to find what a stuffed up :-P thanks

Reply:
fixed with 9177ce2